app/helpers/mountain_view/application_helper.rb

```diff
@@ -0,0 +1,27 @@
+module MountainView
+  module ApplicationHelper
+    def method_missing method, *args, &block
+      if method.to_s.end_with?('_path') or method.to_s.end_with?('_url')
```

Comment:
Use `||` instead of `or`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.